addons/forward/models.py

```diff
@@ -44,7 +44,7 @@ def save(self, request=None, *args, **kwargs):
             else:
                 user = request.user
 
-            self.owner.check_spam(user, {'addons_forward_node_settings__url'}, get_headers_from_request(request))
+            self.owner.do_check_spam(user, {'addons_forward_node_settings__url'}, get_headers_from_request(request), force=True)
```

Comment:
This should not be `do_check_spam`; the signature is `(self, author, author_email, content, request_headers, update=True)`.

Also, `check_spam` handles `SPAM_CHECK_PUBLIC_ONLY`, `do_` does not.